client/overview/task-list/test/index.js

```diff
@@ -44,16 +57,86 @@ describe( 'TaskList', () => {
 		CollapsibleList.mockImplementation( renderChildren );
 		TaskItem.mockImplementation( renderChildren );
```

Comment:
`TaskItem` doesn't have children, so it always renders an empty `div` meaning some of the tests below are invalid.
You might want to use `({ title}) => <div>{ title }</div>` for the TaskItem mock.

Reply:
Nice catch! I added the change in the commit a240722